regulations/static/components/Collapsible.vue

```diff
@@ -16,6 +16,7 @@ export default {
         requestAnimationFrame(() => {
             this.visible = this.state === "expanded";
             this.isVertical = this.direction === "vertical";
+            this.setTabIndex(this.childtag, this.visible);
         });
```

Comment:
Using JS to set tabindex to 0/-1 based on `this.visible`.  I know this component is supposed to be as generic as possible but thought this would be an issue for any content that is a child of this component.  My semi-generic solution was to add an optional prop named `childtag` that can be passed in to component and used to determine which child elements will have tabindex toggled on expand/collapse.

Reply:
Per @whytheplatypus 's review, refactored implementation to add/remove `display: none` to collapse content instead of digging into content and adding/removing tabindexes.  Much cleaner.